styles.css

```diff
@@ -0,0 +1,34 @@
+* {
+  font-family: "Raleway";
+}
```

Comment:
`font` is missing the generic font family. Please add `font-family: Raleway, sans-serif` instead